modules/core/src/components/declarative-ui/xviz-plot.js

```diff
@@ -185,30 +190,35 @@ class XVIZPlotComponent extends PureComponent {
     } = this.props;
 
     const dataProps = this._extractDataProps();
+    const {missingStreams} = this.state;
 
     return (
-      <MetricCard
-        title={title}
-        description={description}
-        style={style}
-        isLoading={dataProps.isLoading}
-      >
-        <MetricChart
-          {...dataProps}
-          getColor={getColor}
-          highlightX={0}
-          width={width}
-          height={height}
-          style={style}
-          xTicks={xTicks}
-          yTicks={yTicks}
-          formatXTick={formatXTick}
-          formatYTick={formatYTick}
-          onClick={this._onClick}
-          formatTitle={this._formatTitle}
-          horizontalGridLines={horizontalGridLines}
-          verticalGridLines={verticalGridLines}
-        />
+      <MetricCard title={title} description={description} style={style} isLoading={false}>
+        <>
```

Comment:
Empty tag?

Reply:
It's a shorthand fragment - the "closing" fragment is on line 221.